src/test/resources/application.properties

```diff
@@ -3,5 +3,7 @@ spring.datasource.driverClassName=org.h2.Driver
 spring.datasource.username=sa
 spring.datasource.password=password
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
```

Comment:
not fixed:
`remove properties:
spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
spring.jpa.database-platform=org.hibernate.dialect.MySQLDialect

because they are deprecated and may produce errors

change to validate, you have liquibase for auto creating`
```suggestion
```